frontend/packages/dev-console/src/components/topology/TopologyPage.tsx

```diff
@@ -78,26 +81,49 @@ const TopologyPage: React.FC<Props> = ({
   cheURL,
   serviceBinding,
 }) => {
-  const namespace = match.params.ns;
+  const namespace = match.params.name;
   const application = activeApplication === ALL_APPLICATIONS_KEY ? undefined : activeApplication;
+  const showListView = matchPath(match.path, {
+    path: '*/list',
```

Comment:
Feels weird that this exists in this state. But I suppose it's code inside a component, so it's not like it can bleed to the rest of the app. It can only handle situations where it is mounted.

Reply:
It's like using a nested route. But since we need to do more than just render a component, I used `matchPath` which uses the same logic as route.